cmd/cerbosctl/get/internal/policy/policy.go

```diff
@@ -51,8 +50,8 @@ func List(k *kong.Kong, c client.AdminClient, filters *flagset.Filters, format *
 	fd := newFilterDef(kind, filters.Name, filters.Version, filters.IncludeDisabled)
 
 	for idx := range policyIds {
-		if idx%internal.MaxIDPerReq == 0 {
-			idxEnd := internal.MinInt(idx+internal.MaxIDPerReq, len(policyIds))
+		if idx%client.MaxIDPerReq == 0 {
```

Comment:
Oh we still have to duplicate this logic in some places, don't we? I wanted to keep it in one place so that it doesn't drift apart (i.e. someone modifies it in one place but forgets to do it in the other places). Looks like that's not possible 🤔

Reply:
Fair enough. Even if we do find a way, that itself would become too complicated to make sense of. Let's leave it as it is.